v2/bindings/csharp/SnapPluginLibCs/SnapPluginLib/SnapPluginLib.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-        <TargetFramework>netstandard1.4</TargetFramework>
+	    <TargetFramework>netstandard2.1</TargetFramework>
```

Comment:
generally we had requirement to use the lowest version possible. There might be a need to integrate with older SW products.

Reply:
ok